<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>960</width>
    <height>640</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Adversarial Attacker</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label_help">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>951</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Choose an image and then click 'Classify' or 'Attack'&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>-5</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_classify">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Classify</string>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="isActivated" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_attack">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Attack</string>
    </property>
    <property name="isActivated" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>540</y>
      <width>891</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="isVisible" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_again">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Try Again</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>911</width>
      <height>411</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>909</width>
       <height>409</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QLabel" name="label_image">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>909</width>
        <height>409</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="isVisible" stdset="0">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>label_help</zorder>
   <zorder>button_attack</zorder>
   <zorder>button_classify</zorder>
   <zorder>label_result</zorder>
   <zorder>button_again</zorder>
   <zorder>scrollArea</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFit_to_window"/>
    <addaction name="actionNormal_size"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFit_to_window">
   <property name="text">
    <string>Fit to window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionNormal_size">
   <property name="text">
    <string>Normal size</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_classify</sender>
   <signal>clicked()</signal>
   <receiver>button_attack</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>303</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>603</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_attack</sender>
   <signal>clicked()</signal>
   <receiver>button_classify</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_again</sender>
   <signal>clicked()</signal>
   <receiver>button_attack</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>892</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_again</sender>
   <signal>clicked()</signal>
   <receiver>button_classify</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>864</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
